scripts/githooks/commit_msg.py

```diff
@@ -42,7 +47,7 @@ def parse_lines(self, lines):
         self.has_subject_body_separator = False
 
         if len(lines) > 1:
-            self.has_subject_body_separator = (len(lines[1].strip()) == 0)
+            self.has_subject_body_separator = len(lines[1].strip()) == 0
```

Comment:
Strange rule to complain about the first approach since it is a lot easier to read. But if that is what it says...

Reply:
I don't remember modifying it, probably that's the black formatter... On the good side, it is not configurable, so we can just give up and live with its minor weirdnesses :)

Reply:
Agree. If this is what either of the tools say out of the box, we just roll with it. Just wasn't sure if that was the case.